pxtlib/initprj.ts

```diff
@@ -1,7 +1,7 @@
 namespace pxt.template {
     export const TS_CONFIG = `{
     "compilerOptions": {
-        "target": "es5",
+        "target": "es2017",
```

Comment:
>es2017 [](start = 19, length = 6)

Wondering: why not es2018 here?

Reply:
Linking https://github.com/microsoft/pxt/pull/7521#discussion_r593599459 for reference, this is the default tsconfig when we create a new github repo in the editor. (this should be esnext though, I missed it in my grep; thanks for pointing it out~)